style.css

```diff
@@ -0,0 +1,155 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  font-family: Courier New, Courier, monospace;
+}
+
+body {
+  background-color: rgb(253, 251, 251);
+}
+
+header {
+  background-color: lightseagreen;
+  top: 0;
+  position: fixed;
+  min-width: 100%;
+  text-align: center;
+  padding: 20px;
+}
+
+header h1 , h2 {
+  color: white;
+  text-shadow: 2px 2px black;
+}
+
+nav {
+  text-align: end;
+  margin-right: 30px;
+}
+
+nav a {
+  text-decoration: none;
+  color: whitesmoke;
+  border-bottom: 1px dashed darkgreen;
+  margin-right: 10px;
+  font-weight: 800;
+}
+
+#minha_foto {
+  min-width: 300px;
+  width: 300px;
+  border-radius: 10rem;
+  margin: -10px;
+  margin-left: 100px;
+}
+
+.photo_section {
+  display: inline-block;
+  min-width: 100%;
+}
+
+#texts {
+  min-width: 100%;
+}
```

Comment:
Alan,
Não sei se é no meu navegador, mas quando eu abri o seu arquivo uma parte da `div #texts` está escondida atras do `header.` Acredito que adicionando um `margin-top` (no meu navegador foi de 100px) de alguns pixels ja resolva. Faz sentido?
```suggestion
   margin-top: 100px
}
```
![Screenshot from 2021-02-27 22-15-36](https://user-images.githubusercontent.com/78616220/109404891-5a549c00-7949-11eb-8549-fc84b069207b.png)

![Screenshot from 2021-02-27 22-41-34](https://user-images.githubusercontent.com/78616220/109405231-0ba90100-794d-11eb-822a-81889fecba14.png)